airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala

```diff
@@ -687,8 +691,15 @@ object LogicalPlan {
     override def sig(config: QuerySignatureConfig): String = {
       s"${joinType.symbol}(${left.sig(config)},${right.sig(config)})"
     }
-    override def inputAttributes: Seq[Attribute] =
-      left.outputAttributes ++ right.outputAttributes
+    override def inputAttributes: Seq[Attribute] = {
+      (getRelationName(left) match {
+        case Some(q: String) => left.outputAttributes.map(_.withQualifier(q))
+        case None            => left.outputAttributes
+      }) ++ (getRelationName(right) match {
+        case Some(q: String) => right.outputAttributes.map(_.withQualifier(q))
+        case None            => right.outputAttributes
+      })
```

Comment:
Propagate relation names for resolving columns that have a table name as prefix.